pkg/intel/metadata/manifest/signature_types.go

```diff
@@ -46,6 +46,7 @@ func NewSignatureData(
 	}
 	switch signAlgo {
 	case AlgRSAPSS:
+		fmt.Println("AlgRSAPSS invoked")
```

Comment:
Garbage

Reply:
Yes I know, thats why the the commit is called: [Print stuff to understand what is going wrong](https://github.com/linuxboot/fiano/pull/363/commits/d74ebcebd92a2aac983ce0dae0728f78cf182def)